bin/dep-audit

```diff
@@ -22,7 +27,11 @@ var argv = require('yargs')
 .argv
 
 if (argv['config-url']) {
-  request(argv['config-url'], function (err, res, body) {
+  request.get({
+    method: 'GET',
```

Comment:
by using request.get method is set to 'GET'. Even if it was something else it would be reset to 'GET'.